plugins/SimulationView/layers3d.shader

```diff
@@ -356,7 +356,10 @@ geometry41core =
             EndPrimitive();
         }
 
-        if ((u_show_starts == 1) && (v_prev_line_type[0] != 1) && (v_line_type[0] == 1)) {
+        if ((u_show_starts == 1) && (
+            ((v_prev_line_type[0] != 1) && (v_line_type[0] == 1)) ||
+            ((v_prev_line_type[0] != 4) && (v_line_type[0] == 4))
+            )) {
```

Comment:
Why is this shader code added to the PR, seems unrelated?

Reply:
This shader code is for showing the start of support. basically showing seams in support

Reply:
I'm a bit puzzled to why this is needed all of a sudden?

Reply:
Since we added support seam, it is only fair that we show them in front end.